_drafts/release-workflows.md

```diff
@@ -0,0 +1,309 @@
+
+##### Pre-releases configuration
+
+```json
+{
+  "release": {
+    "branches": [
+      "master",
+      {"branch": "beta", "prerelease": true},
+      {"branch": "alpha", "prerelease": true},
+    ]
+  }
+}
+```
+
+#### Configuration
+
+##### Format
+
+Each branch is either a `String` or an `Object` with the following properties:
+
+| Option       | Description                                                                                                                                                             |
+|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `branch`     | **Require** Git branch name                                                                                                                                             |
+| `range`      | Accepted version range for to release from this branch. If defined the branch will be considered a LTS branch.                                                          |
+| `channel`    | Channel on which to release. Ignored for release targets that don't support channels.                                                                                   |
+| `prerelease` | Version of the pre-release done from this branch. Must be formatted `<version>-<tag>`. The version is required, the `tag` is optional. Examples: `4.0.0-beta`, `4.0.0`. |
+
+**The order in which the branches are defines is meaningful, and it is used to determine the branch type and which release can be done from each branch.**
+
+##### Branches naming convention
+
+In a branch is defined as a `String` `semantic-release` will automatically determine the type of branch as follow:
+- If the branch name is a valid semver range (Only the format `1.x.x` or `1.x` are valid Git reference names) the branch will be considered a LTS branch, with its range based on the branch name, and release on the channel with the same name
+- The first branch in the `branches` `Array` after the LTS branches is considered the default branch (usually `master`).
+- Each branches in the `branches` `Array` after the default branch are considered future branches.
+
+For example:
+```json
+{
+  "release": {
+    "branches": [
+      "1.x.x",
+      "master",
+      "next",
+      {"branch": "alpha", "prerelease": true}
+    ]
+  }
+}
+```
+is equivalent to:
+```json
+{
+  "release": {
+    "branches": [
+      {"branch": "1.x.x", "range": "1.x.x", "channel": "latest"},
+      {"branch": "master", "channel": "latest"},
```

Comment:
Can two branches refer to the same _channel_. I guess if the _head_ commit is the same, then it's technically the same code (and therefore, the `latest` channel would be pointing to the same artifact, regardless of whether it was generated on the `1.x.x` branch, or the `master` branch)?

Reply:
Actually, a reference in the documentation below mentioned that an LTS branch cannot overlap with the _default_ branch. Therefore, the commits would not be identical.